docs/source/user.rst

```diff
@@ -0,0 +1,254 @@
+User's guide
+============
+
+The purpose of this document is to walk the mead maker through all the
+major operations required to use the software.  Throughout this
+document, this role is referred to as the producer.  This is in
+contrast to the consumer, who plays a significant role in the
+distribution section.
+
+Setup
+-----
+
+``pyment`` is like any other Django app in that it must be deployed in
+order to be useful.  Deployment specifics are beyond the scope of this
+document, but references such as `How to deploy with WSGI
+<https://docs.djangoproject.com/en/1.8/howto/deployment/wsgi/>`_ and
+tutorials such as `How To Deploy a Local Django App to a VPS
+<https://www.digitalocean.com/community/tutorials/how-to-deploy-a-local-django-app-to-a-vps>`_
+are available on the internet.  At the very minimum, the producer will
+need a Django user account in order to access the admin site to
+perform many of the operations listed below.  For more information
+about the admin site, including how to use it, please see `Chapter 6:
+The Django Admin Site
+<http://www.djangobook.com/en/2.0/chapter06.html>`_.
+
+Production
+----------
+
+.. warning:: It is essential to remember that there are many ways to
+	     brew mead, and that this software was written to suit the
+	     particular brewing :doc:`process <brewprocess>` used by
+	     its author.  That being said, other brewers will likely
+	     find this software useful, and :doc:`pull requests <dev>`
+	     are cheerfully encouraged.
+
+This section focuses on the basics of mead production: ingredients,
+recipes, batches, and products.  A discussion of samples is also
+included.  Each of these models is part of the ``meadery`` app.
```

Comment:
In this context, "models" refers to the internal components of the software which describe the objects documented in this section of code.  I can remove this line from the documentation without losing useful content and without adding confusion.